src/server/v2.0/handler/project_metadata.go

```diff
@@ -155,6 +155,12 @@ func (p *projectMetadataAPI) validate(metas map[string]string) (map[string]strin
 			return nil, errors.New(nil).WithCode(errors.BadRequestCode).WithMessage("invalid value: %s", value)
 		}
 		metas[proModels.ProMetaSeverity] = strings.ToLower(severity.String())
+	case proModels.ProMetaProxySpeed:
+		v, err := strconv.ParseInt(value, 10, 32)
```

Comment:
Only proxy cache project can have a speed limit, we should have a check on this

Reply:
> Only proxy cache project can have a speed limit, we should have a check on this

this `func (p *projectMetadataAPI) validate() () {}` function merely serves the purpose of checking the metadata key is in the defined list when adding/updating `ProjectMetadatas` disregarding its relationship with project.

Additionally, `params.Project.Metadata.EnableContentTrust` will be set to nil for proxy cache project, while the current logic of `validate()` does not checking proxy-cache either.  

https://github.com/goharbor/harbor/blob/main/src/server/v2.0/handler/project.go#L550-L554